ddsrouter_yaml/test/unittest/configuration/YamlReaderConfigurationTest.cpp

```diff
@@ -140,51 +139,50 @@ TEST(YamlReaderConfigurationTest, get_ddsrouter_configuration_no_version)
         utils::Formatter error_msg;
         ASSERT_TRUE(configuration_result.is_valid(error_msg)) << error_msg;
     }
+}
 
-    // trivial configuration of v2.0 fails
+/**
+ * Test error in version tag
+ *
+ * CASES:
+ * - not existing version
+ * - get wrongly defined yaml with default version (v3.0)
+ */
+TEST(YamlReaderConfigurationTest, version_negative_cases)
+{
+    // not existing version
     {
+        // trivial configuration
         const char* yml_configuration =
                 R"(
+            version: v0.0
             participants:
               - name: "P1"
                 kind: "echo"
               - name: "P2"
-                kind: echo"
+                kind: "echo"
             )";
 
         Yaml yml = YAML::Load(yml_configuration);
 
-        // Load configuration and check is not valid
+        // Load configuration
         ASSERT_THROW(
             ddsrouter::yaml::YamlReaderConfiguration::load_ddsrouter_configuration(yml),
             utils::ConfigurationException);
     }
-}
-
-/**
- * Test error in version tag
- *
- * CASES:
- * - not existing version
- */
-TEST(YamlReaderConfigurationTest, version_negative_cases)
-{
-    // not existing version
+    // trivial configuration of v3.0 is not correct
```

Comment:
This comment is not correct, or this test does not refer to such comment

Reply:
this test still does not make sense. However it does not make any harm, so let's leave it